spec/pubid_iso/identifier_spec.rb

```diff
@@ -83,4 +83,25 @@
 
     it_behaves_like "converts pubid to urn"
   end
+
+  context "ISO 21143.2" do
+    let(:pubid) { "ISO 21143.2" }
+    let(:urn) { "urn:iso:std:iso:21143" }
+
+    it_behaves_like "converts pubid to urn"
+  end
+
+  context "ISO/FDIS 21420.2" do
+    let(:pubid) { "ISO/FDIS 21420.2" }
+    let(:urn) { "urn:iso:std:iso:21420:stage-50.00.v2" }
+
+    it_behaves_like "converts pubid to urn"
+  end
+
+  context "ISO/TR 30406:2017" do
+    let(:pubid) { "ISO/TR 30406:2017" }
+    let(:urn) { "urn:iso:std:iso:tr:30406" }
+
+    it_behaves_like "converts pubid to urn"
+  end
```

Comment:
@ronaldtse could you confirm if that's correct here?
I don't include iteration number for ISO/TR 30406:2017 because according to RFC we need to have stage together with iteration number, but we don't have it here.

Reply:
@mico "ISO/FDIS 2142.2" => `urn:iso:std:iso:21420:stage-50.00.v2` is correct.